Lib/LinkedIn/LinkedInApi.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+
+use GuzzleHttp\Client;
+
+class LinkedInApi {
```

Comment:
Why not use an off the shelf API library? First result google: https://github.com/ashwinks/PHP-LinkedIn-SDK